package.json

```diff
@@ -24,16 +24,16 @@
   ],
   "description": "Official Sentry SDK for Cordova",
   "dependencies": {
-    "@sentry/browser": "7.34.0",
-    "@sentry/core": "7.34.0",
-    "@sentry/hub": "7.34.0",
-    "@sentry/types": "7.34.0",
-    "@sentry/utils": "7.34.0",
-    "@sentry/wizard": "^3.21.0"
+    "@sentry/browser": "7.119.0",
```

Comment:
Let do one more bump to include the latest security patch. 

https://github.com/getsentry/sentry-javascript/releases/tag/7.119.1